DWIConvert/DWIConverter.cxx

```diff
@@ -82,9 +87,9 @@ Volume4DType::Pointer DWIConverter::OrientForFSLConventions( const bool toFSL)
   static const double FSLDesiredDirectionFlipsWRTLPS[4] = {1,-1,1,1};
   static const double DicomDesiredDirectionFlipsWRTLPS[4] = {1,1,1,1};
   this->ConvertBVectorsToIdentityMeasurementFrame();
-  this->ConvertToMutipleBValuesUnitScaledBVectors();
-
-
+  if (toFSL){
```

Comment:
This code seems wrong.  it should only be used when writing out FSL formatted files, and should be placed in in the FSL writing part of the code.